content/news/2022-07-29-bevy-0.8/index.md

```diff
@@ -0,0 +1,1131 @@
+* "Serialize" logic is now implemented using [`TypeRegistry`] type data like other reflected trait logic, rather than being hard-coded into [`Reflect`] impls. (`@jakobhellermann`)
+
+## Hierarchy Commands
+
+<div class="release-feature-authors">authors: @james7132</div>
+
+Bevy's entity hierarchy system is based on two components: [`Parent`] (which points to an entity's parent) and [`Children`] (which points to a list of the entity's children). This separation is important, as it makes it easy and cheap to query for "hierarchy roots":
+
+```rust
+fn system(roots: Query<Entity, Without<Parent>>) { }
+```
+
+In past versions of Bevy, we built a complex system to "maintain" the integrity of the hierarchy. As [`Parent`] / [`Children`] components were added/removed/changed for an entity, we would do our best to sync everything up across the hierarchy.
+
+However this meant that for a given point in time, the hierarchy could be "out of sync" and incorrect.
+
+Our solution to this problem is to remove the deferred "hierarchy maintenance system" in favor of making hierarchy changes "transactional". Hierarchy changes are now done via transactional [`Commands`], and directly modifying the component fields individually is no longer possible. This ensures that for a given point in time, the hierarchy is "correct".
+
+For most Bevy developers this is a non-breaking change, as most hierarchies are already constructed using `with_children` commands: 
+
+```rust
+commands
+    .spawn_bundle(SpatialBundle::default())
+    .with_children(|parent| {
+        parent.spawn_bundle(SpriteBundle {
+            texture: player_texture,
+            ..default()
+        });
+        parent.spawn_bundle(SpriteBundle {
+            texture: hat_texture,
+            ..default()
+        });
+    });
+```
+
+However for logic that adds/removes child entities from parents at runtime, the following commands must be used:
+```rust
+// removes the given children from the parent
+commands.entity(some_parent).remove_children(&[child1, child2]);
+// pushes the given children to the "end" of the parent's Children list
+commands.entity(some_parent).push_children(&[child3, child4]);
+// inserts the given children into the parent's Children list at the given index 
+commands.entity(some_parent).insert_children(1, &[child5]);
+``` 
+
+We've also added [`HierarchyEvent`], which makes it possible for developers to track changes in the hierarchy.
+
+There are still a couple of small holes to close, but staying on the "happy path" is much easier now:
+* removing only one of the components is possible (although heavily discouraged)
+* adding default values of only one of the components manually is still possible (although heavily discouraged)
```

Comment:
```suggestion
* removing only one of the components is possible (although heavily discouraged);
* adding default values of only one of the components manually is still possible (although heavily discouraged).
```

Reply:
I personally think the bulleted list style already serves as punctuation and that trying to also treat this as one "normal" complete sentence actual makes this less clear